Directory.Build.props

```diff
@@ -12,13 +12,22 @@
   <PropertyGroup>
     <UseFullSDKPaths>true</UseFullSDKPaths>
     <MTGOSDKCoreTFM>netstandard2.0</MTGOSDKCoreTFM>
+    <MTGOSDKCoreTarget>.NETStandard2.0</MTGOSDKCoreTarget>
     <MTGOSDKCorePath>$(SolutionDir)\MTGOSDK\bin\$(Configuration)\$(MTGOSDKCoreTFM)</MTGOSDKCorePath>
+    <PackageOutputPath>$(PublishDir)</PackageOutputPath>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <RestoreLockedMode>true</RestoreLockedMode>
-    <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
-    <RestoreUseStaticGraphEvaluation>true</RestoreUseStaticGraphEvaluation>
+  <!-- Continuous Integration -->
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
+    <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
+  </PropertyGroup>
+
+  <!-- Publishing: Build a separate snupkg symbol package for NuGet. -->
+  <PropertyGroup Condition="'$(CreateSymbolPackage)' == 'true'">
+    <IncludeSymbols>True</IncludeSymbols>
+    <DebugSymbols>True</DebugSymbols>
+    <DebugType>portable</DebugType>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
   </PropertyGroup>
```

Comment:
I've left snupkg generation under a `CreateSymbolPackage` property in case this issue gets resolved. For now, this is incompatible with ILRepack's debug symbol handling.